corehq/apps/geospatial/static/geospatial/js/geospatial_map.js

```diff
@@ -288,6 +288,20 @@ hqDefine("geospatial/js/geospatial_map", [
         self.hasErrors = ko.observable(false);
         self.selectedLocation = null;
 
+        self.setUserFiltersFromUrl = function () {
+            const shouldShowUsers = utils.fetchQueryParam(SHOW_USERS_QUERY_PARAM) || false;
+            self.shouldShowUsers(shouldShowUsers);
+            const userLocationId = utils.fetchQueryParam(USER_LOCATION_ID_QUERY_PARAM);
+            self.selectedLocation = userLocationId;
```

Comment:
nit: could be moved to below if block i.e set only if `userLocationId` is available